crates/ironrdp-pdu/src/rdp/headers.rs

```diff
@@ -518,18 +518,24 @@ bitflags! {
 ///
 /// [2.2.3.1]: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/8a29971a-df3c-48da-add2-8ed9a05edc89
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct ServerDeactivateAll;
+pub struct ServerDeactivateAll {}
```

Comment:
style (non-blocking): The style used across the codebase is the shorthand style (without `{}`)

Reply:
Oops, didn't mean to change that: 1de16664c6c7e90b0f9c5d9c3efc716f9bdbd631